src/data/fintype/basic.lean

```diff
@@ -231,6 +231,9 @@ if hα : nonempty α then by letI := classical.inhabited_of_nonempty hα;
   exact of_surjective (inv_fun f) (inv_fun_surjective H)
 else ⟨∅, λ x, (hα ⟨x⟩).elim⟩
 
+noncomputable instance subtype_of_fintype [fintype α] (p : α → Prop) : fintype (subtype p) :=
+fintype.of_injective coe subtype.coe_injective
+
```

Comment:
xref #5943 - this instance is a duplicate of `subtype.fintype` (in the presence of `classical.dec_pred` anyway)